src/provider.c

```diff
@@ -526,6 +526,10 @@ static int refresh_slots(P11PROV_CTX *ctx)
     return ret;
 }
 
+#if defined(RTLD_DEEPBIND)
```

Comment:
This should be !defined

Reply:
I swear I typed the `!` and it disappeared on its own :)